roles/create_pr/README.md

```diff
@@ -13,6 +13,7 @@ partner_email                       | undefined | true/false                  |
 target_repository                   | undefined | true                        | GitHub repository where the pull request will be created.
 product_type                        | undefined | true                        | Product type. Either "helmchart" or "operator".
 publish_chart                       | true      | false                       | For helm charts, disable adding the Chart as part of the pull request.
+git_protocol                        | ssh       | false                       | Protocol for GitHub communication (ssh/https)
```

Comment:
don't you already have this fragment from the `target_repository` variable? you can infer it by doing something like `git_protocol: "{{ target_repository | urlsplit('scheme') }}"`